.github/workflows/build.yml

```diff
@@ -38,6 +38,10 @@ jobs:
       coverage: ${{ matrix.os == 'ubuntu-latest' && matrix.node-version == 14 }}
       NODE_VERSION: ${{ matrix.node-version }}
 
+    permissions:
+      actions: write
```

Comment:
I think this is required for the publish coverage artifact step below.

Reply:
no, i don't think so